pom.xml

```diff
@@ -294,45 +294,6 @@
       <activation>
         <jdk>[1.8,)</jdk>
       </activation>
-      <build>
```

Comment:
Probably better to remove the full 'java8-enable-like-dependencies' profile because it won't make sense now.